sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -795,12 +797,18 @@ aggregates AS (
       scalar_parent_devtools_toolbar_eyedropper_opened
     ) AS scalar_parent_devtools_toolbar_eyedropper_opened_sum,
     NULL AS scalar_parent_dom_contentprocess_troubled_due_to_memory_sum, -- deprecated
+    mozfun.stats.mode_last(
+      ARRAY_AGG(scalar_parent_os_environment_launch_method ORDER BY submission_timestamp)
+    ) AS scalar_parent_os_environment_launch_method,
```

Comment:
Su indicated this should be rolled up as an array. The way this field would be analyzed is the number of clients launching Fx with each launch method; so a single client could plausibly have multiple on the same day.